enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -525,8 +525,10 @@ class SubsidyAccessPolicyCreditsAvailableResponseSerializer(SubsidyAccessPolicyR
 
     @extend_schema_field(serializers.IntegerField)
     def get_remaining_balance_per_user(self, obj):
-        lms_user_id = self.context.get('lms_user_id')
-        return obj.remaining_balance_per_user(lms_user_id=lms_user_id)
+        if hasattr(obj, 'remaining_balance_per_user'):
+            lms_user_id = self.context.get('lms_user_id')
+            return obj.remaining_balance_per_user(lms_user_id=lms_user_id)
+        return None
```

Comment:
Serialization of assignment-based policies in `credits_available` blows up without this.